Assets/Scripts/Utility/AssetInjection/Components/CustomDoor.cs

```diff
@@ -24,20 +24,29 @@ namespace DaggerfallWorkshop.Utility.AssetInjection
     [RequireComponent(typeof(BoxCollider))]
     public class CustomDoor : MonoBehaviour
     {
+        [Tooltip("Type of door that is being defined. None means building textures are used to determine this.")]
+        public DoorTypes doorType = DoorTypes.None;
+        
+        [Tooltip("Number of BoxColliders alongside script that will be used as door activators")]
+        public int numColliders = 1;
+    
```

Comment:
A minor syntax note: please use pascal case for public members and place them after `DoorTrigger`.

Reply:
Done.